tests/test_plot.py

```diff
@@ -1,22 +1,23 @@
+import unittest
 from probability_calculator.density import DiscreteDensity
-from probability_calculator.plot import getPlotData, kernelDensityEstimation
+from probability_calculator.plot import get_plot_data, kernel_density_estimation
 
 
-class TestPlot():
+class TestPlot(unittest.TestCase):
     def test_getPlotData(self):
         d = DiscreteDensity(outcomes=[
-            {"value": 1, "prob": 1./6},
-            {"value": 2, "prob": 1./3},
-            {"value": 1, "prob": 1./6},
-            {"value": 3, "prob": 1./3}
+            {"value": 1, "probability": 1. / 6},
```

Comment:
I understand that prob is maybe not a good idea. However, probability is very long in a place where the meaning is total clear. How about just use `p`?

Reply:
yeah :) no strong feelings about that...